submit draft July TLC minutes review and approval (#300)
carpentries/trainers: leadership_minutes/2024/2024-07-11-leadership-minutes.md

Trainers Leadership July 2024-07-11

Attendance
Present: Jon, Liz, Nathaniel
Apologies: Sher!, Intekhab, Annajiat
Notetaker: Jon

## Agenda and notes

1.  Update on action items
    a.  Nathaniel will look up info from last year's trainer active status renewal process. By the next meeting.
	    1. From 5 Dec 2023 notes on renewal:
		    1.  Most valuable alternative activities included Trainers Leadership and IT curriculum maintenance.
		    2.  4 demos could substitute for 1 Instructor Training.

> 		    3. 2 Instructor Training could cover other requirements (certainly all demos).

Reviewer: ```suggestion
		    3. Two Instructor Training events could cover other requirements (certainly all demos).
```

		    4. Otherwise, it will be hard to get away from a case-by-case review if we want to keep the "alternative activities" option for sustaining active status.

> 		    5. Number of people who have done demos or workshops that weren't included in their summary was alarmingly high. Does this reflect a problem?

Reviewer: ```suggestion
		    5. A number of people who have done demos or workshops that weren't included in their summary was alarmingly high. Does this reflect a problem?
```

> 			    1. Demos are recorded in AMY when host submits results (maybe? Nathaniel will check with Core Team).

Reviewer: ```suggestion
			    1. Demo hosts are recorded in calendly and demo host form, with records maintained and provided for renewal process by Maneesha (not recorded in AMY)
```
